lib/measures/tbd/resources/psi.rb

```diff
@@ -257,7 +257,7 @@ def deratableLayer(construction)
       m                 = m.to_StandardOpaqueMaterial.get
       k                 = m.thermalConductivity
       d                 = m.thickness
-      if d < 0.003 || k < 3.0 || d / k < r
+      if d < 0.003 || k > 3.0 || d / k < r
```

Comment:
Typo in thermal conductivity (k) check. Materials with k > 0.003 should be skipped (i.e. not derated), as per EnergyPlus recommendations. The typo was ensuring the opposite.